litho-core/src/main/java/com/facebook/litho/MountState.java

```diff
@@ -1569,11 +1568,11 @@ private static void unsetViewAttributes(MountItem item, boolean isHostView) {
     final ViewNodeInfo viewNodeInfo = item.getViewNodeInfo();
     if (viewNodeInfo != null) {
       unsetViewStateListAnimator(view, viewNodeInfo);
+      unsetViewBackground(view, viewNodeInfo);
       // Host view doesn't set its own padding, but gets absolute positions for inner content from
-      // Yoga. Also bg/fg is used as separate drawables instead of using View's bg/fg attribute.
+      // Yoga. Also fg is used as separate drawables instead of using View's fg attribute.
```

Comment:
It would be nice to implement foreground support for Hosts so that we can treat foreground and background in the same way. This is not something we need to do in this PR

Reply:
Sure. I will do foreground changes in another PR. 
I haven't tested foreground completely.